guides/common/modules/proc_installing-and-configuring-puppet-agent-manually.adoc

```diff
@@ -3,32 +3,25 @@
 
 Use this procedure to install and configure the Puppet agent on a host manually.
 
-ifndef::managing-configurations-puppet[]
-ifdef::satellite[]
-For more information about Puppet, see {ManagingConfigurationsPuppetDocURL}[Managing Configurations Using Puppet Integration in {Project}].
-endif::[]
-endif::[]
-
 .Prerequisites
 * The host must have a Puppet environment assigned to it.
 ifdef::satellite[]
-* The {ProjectName} Client repository must be enabled and synchronized to {ProjectServer}, and enabled on the host.
-The `puppet-agent` rpm is available within the following repositories:
-** {Team} {project-client-name} for RHEL 7 Server RPMs x86_64 {endash} `{RepoRHEL7ServerSatelliteToolsProductVersion}`
-** {Team} {project-client-name} for RHEL 8 Server RPMs <arch> {endash} `{RepoRHEL8ServerSatelliteToolsProductVersion}`
-** {Team} {project-client-name} for RHEL 9 Server RPMs <arch> {endash} `{RepoRHEL9ServerSatelliteToolsProductVersion}`
+* The *{project-client-name}* repository must be enabled and synchronized to {ProjectServer}, and enabled on the host.
+For more information, see {ContentManagementDocURL}Importing_Content_content-management[Importing Content] in _{ContentManagementDocTitle}_.
 endif::[]
-ifdef::orcharhino[]
-* The {Team} {project-client-name} repository must be enabled and synchronized to {ProjectServer}, and enabled on the host.
-endif::[]
-ifndef::satellite,orcharhino[]
+ifndef::satellite[]
 * Ensure a repository containing the Puppet agent is enabled on the host, for example https://apt.puppet.com/[apt.puppet.com] or https://yum.puppet.com/[yum.puppet.com].
 endif::[]
 
 .Procedure
 . Log in to the host as the `root` user.
+ifdef::satellite[]
+. Install the Puppet agent package:
+endif::[]
+ifndef::satellite[]
 . Install the Puppet agent package.
 * On hosts running {EL}:
+endif::[]
 +
 [options="nowrap", subs="+quotes,verbatim,attributes"]
 ----
```

Comment:
Should there be `{package-install}` used below?
Also, on 3.1 it will probably need to be split for EL 7 and 8.